html/9c0e05a2-4481-4e97-8081-77753f9aebb1.html

```diff
@@ -49,76 +50,99 @@ <h4>Student Activity </h4>
+Subtract 5 from each side.<br>
+\(y+5-5+3x=-3x+3x+6-5\)</p>
+<p><strong>Step 4</strong> - Simplify.<br>
+Make sure that the \(x\) term is in front of the \(y\) term on the left side.<br>
+\(3x +y =1\)</p>
+<ol class="os-raise-noindent" start="8">
   <li> Write the equation \(y = \frac{1}{3}x-4\) in standard form. </li>
 </ol>
 <p><strong>Answer:</strong> \(x -3y=12\)</p>
-<p><strong>Step 1</strong>- Get rid of the fractions in the equation.</p>
-<p>Multiply every term by 3.</p>
-<p>\(3y = 3(\frac{1}{3}x-4)\)</p>
-<p><strong>Step 2</strong> - Distribute and simplify.</p>
-<p>\(3y=x -12\)</p>
-<p><strong>Step 3 </strong>- Rearrange terms so the \(x\) term and the \(y\) term are both on the left side of the equation.</p>
-<p>Subtract \(x\) from both sides.</p>
-<p>\(3y - x = x - x -12\)</p>
-<p><strong>Step 4</strong> - Simplify.</p>
-<p>\(3y - x = -12\)</p>
-<p><strong>Step 5</strong> - Write the equation so the \(x\) term is first.</p>
-<p>\(-x + 3y = -12\)</p>
-<p><strong>Step 6</strong> - Multiply each term by -1, so as not to lead with a negative.</p>
-<p>\(-1(-x + 3y = -12)\)</p>
-<p>\(x - 3y = 12\)</p>
+
+<p><strong>Step 1</strong>- Get rid of the fractions in the equation.<br>
+Multiply every term by 3.<br>
+\(3y = 3(\frac{1}{3}x-4)\)</p>
+<p><strong>Step 2</strong> - Distribute and simplify.<br>
+\(3y=x -12\)</p>
+<p><strong>Step 3 </strong>- Rearrange terms so the \(x\) term and the \(y\) term are both on the left side of the equation.<br>
+Subtract \(x\) from both sides.<br>
+\(3y - x = x - x -12\)</p>
+<p><strong>Step 4</strong> - Simplify.<br>
+\(3y - x = -12\)</p>
+<p><strong>Step 5</strong> - Write the equation so the \(x\) term is first.<br>
+\(-x + 3y = -12\)</p>
+<p><strong>Step 6</strong> - Multiply each term by -1, so as not to lead with a negative.<br>
+\(-1(-x + 3y = -12)\)<br>
+\(x - 3y = 12\)</p>
+<ol class="os-raise-noindent" start="9">
+    <li>What is the slope of  \(3x + 12y = 24\)? </li>
+</ol>
+<p><strong>Answer:</strong> Compare your answer:</p>
+
+<p>The slope is \(m = -\frac AB\). 
+
+  <br>
+  \(m = -\frac{3}{12}\)<br>
+\(m = -\frac14\)</p>
 <br>
 <h4>Video: Writing Equations in Different Forms</h4>
 <p>Watch the following video to learn more about writing equations in different forms.</p>
 <div class="os-raise-d-flex-nowrap os-raise-justify-content-center">
-  <div class="os-raise-video-container"><iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen frameborder="0" src="https://www.youtube-nocookie.com/embed/0KhxVDBVGRo?si=31FamA-aJe4EvLpH" title="YouTube video player"></iframe></div>
+    <div class="os-raise-video-container">
+        <iframe src="https://youtu.be/0KhxVDBVGRo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen=""></iframe>
```

Comment:
I think this src can be reverted to `https://www.youtube-nocookie.com/embed/0KhxVDBVGRo?si=31FamA-aJe4EvLpH`.